src/main/java/org/motechproject/newebodac/domain/Vaccinee.java

```diff
@@ -69,4 +69,8 @@
   public Vaccinee(UUID id) {
     super(id);
   }
+
+  public void setVaccinee(String vaccinee) {
```

Comment:
where it's used?

Reply:
it is used in ObjectMapper. That's for the special case, when vaccineeId is set with 'vaccinee' field (taken from field config) in RELATION Visit <- Vaccinee

Reply:
it is being used by ObjectMapper. It is being used in special case when 'vaccineeId' is being set with 'vaccinee' from FieldConfig in Relation Visit<-Vaccinee